source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -3,278 +3,528 @@
+   edit, or skip and keep going to the next bundle. If you skip a file, a backup
+   of the original file is saved with the ``.orig`` suffix. Because mixer always
+   checks your local bundles first, edited copies of an upstream bundle will
+   always take precedence over their upstream counterpart.
+
+   This same command is used to create a totally **new bundle**: if the bundle
+   name you specify is not found upstream, a blank template is generated in
+   :file:`local-bundles` with the correct filename. The editor is again launched
+   for you to fill out the bundle, and validation is performed on exiting. Add
+   your package name(s) in the bundle definition file to tell it what package(s)
+   must be installed as part of that bundle.
+
+   Mixer will do basic **validation** on all bundles when they are used
+   throughout the system: it will check that the bundle syntax is valid and can
+   be parsed, and that the bundle file has a valid name. If you would like to
+   manually run this validation on a bundle, you can run:
+
+   .. code-block:: bash
+
+      # mixer bundle validate bundle1 [bundle2...]
+
+   This command has an optional :option:`--strict` flag, which additionally
+   checks that the rest of the bundle header fields can be parsed and are
+   non-empty, and that the bundle header ``Title`` field and the bundle filename
+   match.
+
+   .. note::
+         If you initialized your workspace to be tracked as a Git repository
+         (:command:`mixer init --git`), you may find it useful to apply a git
+         commit after modifying what bundles are in your Mix Bundle List or
+         editing a bundle definition file. All of the above :command:`mixer 
+         bundle` commands support an optional :option:`--git` flag that will
+         automatically apply a git commit when they are finished.
+
+#. **Build the bundle chroots**. To build all of the ``chroots`` that are
+   based on the bundles you defined, run the following command in your
+   workspace:
+
+   .. code-block:: bash
+
+    # sudo mixer build chroots
+
+   If you have many bundles in your mix, this step might take some
+   time.
+
+   By default, mixer will use the legacy chroot-builder. In this mode, the
+   bundle definition files for the bundles in your mix will be automatically
+   gathered into a :file:`mix-bundles` directory in the location specified by
+   ``BUNDLE_DIR`` in your :file:`builder.conf`. **Do not edit these files**.
+   Mixer will automatically clear out any contents of this directory before
+   populating it on-the-fly as chroots are built.
+
+   Mixer now has a new chroot-builder built into the mixer tool itself. While
+   this is currently an experimental feature, you can (and should) use the new
+   chroot-builder by passing the :option:`--new-chroots` flag. The legacy 
+   chroot-builder will soon be deprecated, and mixer will use the new version
+   automatically.
```

Comment:
@kevincwells, regarding:
+   Mixer now has a new chroot-builder built into the mixer tool itself. While
+   this is currently an experimental feature, you can (and should) use the new
+   chroot-builder by passing the :option:`--new-chroots` flag. The legacy 
+   chroot-builder will soon be deprecated, and mixer will use the new version
+   automatically.

So if you don't specify the --new-chroots flag, which chroot builder is using by default?  The new experimental one or the legacy?  And if it's the new one by default, why "should" you specify the flag?

Reply:
>So if you don't specify the --new-chroots flag, which chroot builder is using by default? The new experimental one or the legacy?

A couple lines earlier (L381), the paragraph starts with "By default, mixer will use the legacy chroot-builder"